pystac/utils.py

```diff
@@ -10,11 +11,27 @@
 
 from pystac.errors import RequiredPropertyMissing
 
+HREF = Union[str, os.PathLike]
```

Comment:
```suggestion
Href = Union[str, os.PathLike]
```

Since it behaves like a type/class, would `Href` be a more appropriate name?

Reply:
Maybe? That's actually there to prevent a circular import from link.py I'll look into a more correct way to get this done.

Reply:
@gadomski I eliminated the double definition of `HREF`, which is used in a number of files. Let me know if you'd still like to rename `HREF` to `Href`.

Reply:
Eh, if it was already `HREF` then let's keep it. Thanks!